From 109705375359931b906228a84361fd75269b028e Mon Sep 17 00:00:00 2001
From: Andy Shevchenko <andriy.shevchenko@linux.intel.com>
Date: Thu, 27 Feb 2020 16:41:10 +0200
Subject: [PATCH 2/7] serial: ns16550: Provide UART base clock speed in
 ->getinfo()

Some callers may need the UART base clock speed value.
Provide it in the ->getinfo() callback.

Signed-off-by: Andy Shevchenko <andriy.shevchenko@linux.intel.com>
---
 drivers/serial/ns16550.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/serial/ns16550.c b/drivers/serial/ns16550.c
index a2f1b35629..932d03ca79 100644
--- a/drivers/serial/ns16550.c
+++ b/drivers/serial/ns16550.c
@@ -476,6 +476,7 @@ static int ns16550_serial_getinfo(struct udevice *dev,
 	info->reg_width = plat->reg_width;
 	info->reg_shift = plat->reg_shift;
 	info->reg_offset = plat->reg_offset;
+	info->clock = plat->clock;
 	return 0;
 }
 
-- 
2.20.1

